src/olympia/hero/views.py

```diff
@@ -25,6 +32,11 @@ def list(self, request, *args, **kwargs):
         # Simulate pagination-like results, without actual pagination.
         return Response({'results': serializer.data})
 
+    @classonlymethod
+    def as_view(cls, actions=None, **initkwargs):
+        view = super().as_view(actions=actions, **initkwargs)
+        return non_atomic_requests(view)
```

Comment:
With our `NonAtomicRequestsForSafeHttpMethodsMiddleware` that shouldn't be necessary for `GET` operations anymore. Did you notice extra the transaction statements when calling the API ?

Reply:
I did, yeah.  There was a savepoint at the start and end.

Reply:
I think now that we have the middleware those only happens in tests, because of the way tests are wrapped in a transaction. But I guess that's fine to keep.